doc/rados/operations/pools.rst

```diff
@@ -557,7 +557,7 @@ You may set values for the following keys:
 
 .. describe:: scrub_min_interval
    
-   :Description: Sets the minimum interval (in seconds) between successive shallow / light scrubs of the pool's PGs when the load is low. If the default value of ``0`` is in effect, then the value of ``osd_scrub_min_interval`` from central config is used.
+   :Description: Sets the minimum interval (in seconds) between successive shallow / light scrubs of the pool's PGs. If the default value of ``0`` is in effect, then the value of ``osd_scrub_min_interval`` from central config is used.
```

Comment:
While you're in this file, please

s/is in effect/is set/

The current wording ambiguously sort of implies that there's some external gate to whether the setting is applied.

Reply:
The (existing) sentence tries to convey the following condition:
"if the pool configuration was unchanged from its default (0), then ...".
I'll change it to use something along this line.